docs/requirements.txt

```
@@ -1,5 +1,6 @@
 cirq-superstaq[dev]~=0.5.6
 ipywidgets>=8.0.0
 nbsphinx>=0.9.0
+pandoc~=2.3.0
 qiskit-superstaq[dev]~=0.5.6
 sphinx-rtd-theme>=1.0.0
```

given that building docs is part of our checks, we should probably add `pandoc` and `ipywidgets` to checks-superstaq/requirements.txt (`nbsphinx` and `sphinx-rtd-theme` are already there). then i think this file could just be
```suggestion
cirq-superstaq[dev]~=0.5.6
qiskit-superstaq[dev]~=0.5.6
```